schema/v1.1.0/dataset_config_models_extended.py

```diff
@@ -127,8 +127,8 @@ def check_skip_validation(obj: BaseModel, field_name: str, case_sensitive: bool
                 field_name,
                 field_value,
             )
-            return False
-    return True
+            return True
+    return False
 
```

Comment:
this function now returns whether or not to skip validation (makes more sense when reading it)